industry/foc/Kconfig

```
@@ -58,6 +58,27 @@ config INDUSTRY_FOC_ANGLE_OPENLOOP
 	---help---
 		Enable support for open-loop angle handler
 
+config INDUSTRY_FOC_ANGLE_OBSERVER
+	bool "FOC angle observer handler"
+	default n
+	---help---
+		Enable support for observer angle handler
+
+if INDUSTRY_FOC_ANGLE_OBSERVER
+
+choice
+	prompt "FOC observer type selection"
+
+config INDUSTRY_FOC_OBSERVER_SLID
+	bool "Enable slidmode observer"
+
+config INDUSTRY_FOC_OBSERVER_FLUX
+	bool "Enable fluxlink observer"
+
+endchoice # FOC observer type selection
+
+endif # INDUSTRY_FOC_ANGLE_OBSERVER
+
```

Let's make this like for vel observers:
```suggestion
config INDUSTRY_FOC_ANGLE_OSMO
	bool "Enable slidmode observer"
	default n

config INDUSTRY_FOC_ANGLE_ONFO
	bool "Enable fluxlink observer"
        default n
```